scheds/rust/scx_mitosis/src/bpf/mitosis.bpf.c

```diff
@@ -612,13 +640,37 @@ void BPF_STRUCT_OPS(mitosis_dispatch, s32 cpu, struct task_struct *prev)
 
 	cell = READ_ONCE(cctx->cell);
 
-	if (scx_bpf_dsq_move_to_local(HI_FALLBACK_DSQ))
-		return;
+	bool found = false;
+	u64 min_vtime_dsq;
+	u64 min_vtime;
 
-	if (scx_bpf_dsq_move_to_local(cell))
-		return;
+	struct task_struct *p;
+	bpf_for_each(scx_dsq, p, cell, 0)
+	{
+		min_vtime = p->scx.dsq_vtime;
+		min_vtime_dsq = cell;
+		found = true;
+		break;
+	}
 
-	scx_bpf_dsq_move_to_local(LO_FALLBACK_DSQ);
+	u64 cpu_dsq = pcpu_dsq_id(cpu);
+	bpf_for_each(scx_dsq, p, cpu_dsq, 0)
```

Comment:
Maybe skip the iteration if `found` is `true`? The iteration is relatively expensive involving grabbling and releasing the DSQ lock. Hopefully, in the future, we should have cheaper `scx_bpf_dsq_peek()`.